components/shared/embedder/lib.rs

```diff
@@ -286,11 +282,9 @@ impl Debug for EmbedderMsg {
             EmbedderMsg::SetClipboardContents(..) => write!(f, "SetClipboardContents"),
             EmbedderMsg::SetCursor(..) => write!(f, "SetCursor"),
             EmbedderMsg::NewFavicon(..) => write!(f, "NewFavicon"),
-            EmbedderMsg::HeadParsed(..) => write!(f, "HeadParsed"),
             EmbedderMsg::HistoryChanged(..) => write!(f, "HistoryChanged"),
             EmbedderMsg::SetFullscreenState(..) => write!(f, "SetFullscreenState"),
-            EmbedderMsg::LoadStart(..) => write!(f, "LoadStart"),
-            EmbedderMsg::LoadComplete(..) => write!(f, "LoadComplete"),
+            EmbedderMsg::NotifyLoadStatusChanged(..) => write!(f, "LoadComplete"),
```

Comment:
```suggestion
            EmbedderMsg::NotifyLoadStatusChanged(..) => write!(f, "NotifyLoadStatusChanged"),
```

Reply:
How about matching on the inner enum, so that we can continue seeing the actual event?

Reply:
I'm okay with it, but this probably should also apply to [components/constellation/tracing.rs](https://github.com/servo/servo/pull/35260/files#diff-8692bef9d31feca8b9a01989f1d5d63fa17c4dfecd000f18846de07cecd9bdf6)

Reply:
Okay. I've done this in both places.